11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,63 @@
+  to whitelist (verb):
+    - to allow
+    - to trust
+    - to include
+reading:
+  - text: UK security agency bans blacklist and whitelist for being racist
+    href: https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+  - text: '"Blacklists” and “whitelists": a salutary warning concerning the prevalence of racist language in discussions of predatory publishing'
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+To allow or deny access to a service, a privilege, or recognition.
+
+## Issues
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The term are rooted in racism: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
+
+Using "black" and "white" to determine whether an entity is included or excluded from a group is correlated to the racist idea that Black and White people have been respectively denied and allowed privileges based on the color of their skin. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used casually in the realm of software development, which reinforces the exclusion of Black people from the industry.
+
+<!-- Black/white as a good/bad dichotomy -->
+
+The second issue with these terms is the implication that black is bad and white is good. A "blacklist" often refers to a list of undesirable things (spammy phone numbers, undesired guests, etc.) whereas a "whitelist" refers to a list of virtuous things (important guests, emails we want to receive, etc.) In 1610, the _Douglas Harper Etymology Dictionary_ defined the term "blacklist" as a "list of persons who have incurred suspicion".
+
+From [this NCBI article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/):
+
+> "To compound the issue, it is also striking how often the term “whitelist” is used for a supposedly good, respectable, or safe list of publishers. The racism in such “black is bad, white is good” metaphors is inappropriate and needs to cease."
+
+These terms not only reflect, but they also reinforce and perpetuate a racist culture.
+
+## Usage Tip
+
+Seek to change any existing "blacklist/whitelist" terminology to one of the suggested terms, which have the advantage of being more descriptive and less jargony.
+
+Even if you may not be negatively impacted by this terminology, it can negatively impact other existing and prospective contributors onto your project. The terminology isn't so unique that it's irreplaceable; the lift tends to be trivial in comparison to the potential impact.
```

Comment:
```suggestion
Even if you may not be negatively impacted by this terminology, it can negatively impact other existing and prospective contributors onto your project and spaces. The terminology isn't so unique that it's irreplaceable; the lift tends to be trivial in comparison to the potential impact.
```